NPPESAPI.net/NPPESApiClient.cs

```diff
@@ -1,36 +1,44 @@
-﻿using Forcura.NPPES.Core;
-using Newtonsoft.Json;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Net.Http;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using Forcura.NPPES.Core;
+using Newtonsoft.Json;
```

Comment:
Why we no use Forcura-Serialization?

Reply:
☝🏾  that sounds like something I would type/say lol.  This is a public package for the world, so we draw the line on the dependencies.

Reply:
lol. That make sense.